lms/static/js/spec/financial-assistance/financial_assistance_form_view_spec.js

```diff
@@ -25,7 +25,9 @@ define([
                         placeholder: '',
                         required: true,
                         requiredStr: '',
-                        type: 'select'
+                        type: 'select',
+                        supplementalLink: '',
```

Comment:
Could you make these new fields optional? It is very annoying to have to add them everywhere.